Scripts/GenerateSampleViewSourceCode.swift

```diff
@@ -139,15 +140,26 @@ private let sampleStructs = sampleMetadata
 private let entries = sampleMetadata
     .map { sample in "\(sample.structName)()" }
     .joined(separator: ",\n        ")
+
+private let ARFilteredEntries = sampleMetadata
+    .filter { $0.category != "Augmented Reality" }
+    .map { sample in "\(sample.structName)()" }
+    .joined(separator: ",\n        ")
+
 private let arrayRepresentation = """
-    [
-            \(entries)
-        ]
+    {
     #if targetEnvironment(macCatalyst) || targetEnvironment(simulator)
         // Exclude AR samples from Mac Catalyst and Simulator targets
         // as they don't have camera and sensors available.
-        .filter { $0.category != "Augmented Reality" }
+        [
+            \(ARFilteredEntries)
+        ]
+    #else
+        [
+            \(entries)
+        ]
     #endif
+    }()
```

Comment:
Here's a preview of the new output.

[SamplesApp+Samples.swift.zip](https://github.com/user-attachments/files/17604603/SamplesApp%2BSamples.swift.zip)

It's not great but short of changing the the way `SamplesApp+Samples` is constructed, this is the best short term reformatting I've come up with so far.